AndreiFamishkin/01.html/task3.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+  <HTML>
+	<HEAD>
+		<METAcharset="utf-8">
```

Comment:
where is whitespace between meta and charset?
Could you please checks in all files